docs/_documentations/mdt-vsc-firstproject.md

```diff
@@ -15,23 +15,23 @@ To create your first project, click on the prompt in the **CODEWIND** window
 
 ![](dist/images/createproject.png)
 
-This will prompt you with a list of project types to select from.
+This prompt gives you a list of project types for you to select 
 
 ![](dist/images/listtemplates.png)
 
 Scroll through the list until you see **Node.js Express (Default templates)**
 
 ![](dist/images/nodetemplate.png)
 
-Selecting this template will prompt you to enter a name for your project
+Selecting this template prompts you to enter a name for your project
 
 ![](dist/images/projectname.png)
 
-Enter a name and the choose where to create this on disk (for example myFirstNodeProject)
+Enter a name and then choose where to create this on disk. For example, myFirstNodeProject
```

Comment:
Lines 18, 26, and 30 could all have periods at the end...or maybe a colon instead for lines 18 and 26. Also, "myFirstNodeProject" could be in `code phrase format`.